amcharts3-react.js

```diff
@@ -173,7 +199,7 @@
           break;
 
         default:
-          if (x !== y) {
+          if (!isEqual(x,y)) {
```

Comment:
The type checks above means that the `default` case will only handle simple types like strings.

Therefore, using `isEqual` is unnecessary and inefficient.